releases.md

```diff
@@ -26,6 +26,24 @@ A Gazebo release follows the form "Gazebo Codename", for example Gazebo Acropoli
 | [Blueprint](https://gazebosim.org/docs/blueprint)        | May, 2019 | Dec, 2020 | EOL   |
 | [Acropolis](https://gazebosim.org/docs/acropolis)        | Feb, 2019 | Sep, 2019 | EOL   |
 
+~~~mermaid
```

Comment:
You'll need to add [`sphinxcontrib-mermaid`](https://github.com/mgaitan/sphinxcontrib-mermaid) to requirements.txt. I started working on on a timeline for the release process, but I abandoned the idea. Here's some WIP code that might be useful https://github.com/gazebosim/docs/compare/master...azeey:docs:mermaid